### PR TITLE
motor direction tools: gui_log the direction clicked

### PR DESCRIPTION
**Pull request by nerdCopter**
* requested by @darkmann
* reused existing locale

ignore the missing graphic, it is a side effect of unfinished PWA (vite).

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

File: src/components/EscDshotDirection/EscDshotDirectionMotorDriver.js
Comment (chmelevskij) [inline src/components/EscDshotDirection/EscDshotDirectionMotorDriver.js:141]: I'm a bit confused on what's happening here with all the `+=`

From what I understand all of these states are mutually exclusive aka only one of them can exist at a single point in time. `+=` means that we are appending to the existing string.

Wouldn't it just work with simple assignment `=`?

Then another comment is about usage of `concat`, maybe string literals would be a better choice?

So things like:
```
logString += i18n.getMessage(`motorNumber${motorNumber}`).concat(': ', i18n.getMessage('escDshotDirectionDialog-Open')).concat(' ',i18n.getMessage('escDshotDirectionDialog-CommandNormal'));
```
becomes
```
logString = `${i18n.getMessage(`motorNumber${motorNumber}`)}: ${i18n.getMessage('escDshotDirectionDialog-Open')} ${i18n.getMessage('escDshotDirectionDialog-CommandNormal')}`;
```

Comment (nerdCopter) [reply src/components/EscDshotDirection/EscDshotDirectionMotorDriver.js]: yes, i wanted use `+=` logic, and upon success of prototyping, i forgot to continue such direction.  my next commit solves this with "proper" `+=` usage.

**Review by haslinghuis — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
